TabulateSmarterTestAdminPackage.Common/Validators/MaxLengthValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using TabulateSmarterTestAdminPackage.Common.Enums;
+
+namespace TabulateSmarterTestAdminPackage.Common.Validators
+{
+    public class MaxLengthValidator : Validator
+    {
+        public MaxLengthValidator(ErrorSeverity errorSeverity, object parameter) : base(errorSeverity, parameter)
+        {}
+
+        public override bool IsValid(object value)
+        {
+            return value is string
+                   && Parameter is int
+                   && ((string)value).Length <= (int)Parameter;
+        }
+
+        public override string GetMessage()
+        {
+            return $"[Length<={(Parameter is int ? Parameter.ToString() : string.Empty)}]";
```

Comment:
I took the ToString off. I'll run it through my integration tests and make sure everything is ok